modules/pel/peltool/peltool.py

```diff
@@ -365,6 +371,8 @@ def main():
                         help='Used with -d, only look for files with this extension (e.g. ".pel")')
     parser.add_argument('-x', '--delete', dest='delete', action='store_true',
                         help='Delete original file after parsing')
+    parser.add_argument('-t', '--termination', nargs='?', const=critSysTermSeverity,
```

Comment:
Since the original peltool only looked for the terminating severity with this option, lets keep it like that.

You can add a new option to handle letting the user choose a severity, though make sure to take the input as a hex number like 0x51 since that's how they are listed in the spec and in the output.